### PR TITLE
CLI installer: kubernetes

### PR DESCRIPTION
**Pull request by schoren**
This PR adds support for k8s installer

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: cli/installer/kubernetes.go [hunk 1]
```diff
@@ -21,7 +25,258 @@ var kubernetes = installer{
...
+		)
+	}
+
+	execCmd(helmCmd(conf, cmd...), ui)
+
+	psql := "host=tracetest-postgresql user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+	c := getTracetestConfigFileContents(psql, ui, conf)
+	ttc := createTmpFile("tracetest-config", string(c), ui)
+	defer os.Remove(ttc.Name())
+	execCmd(
```
Comment (mathnogueira) [inline cli/installer/kubernetes.go:70]: I saw you have a `execCmdIgnoreErrors` function. And I see you also use `execCmd` without assigning the returned error. Do you really need that `execCmdIgnoreErrors` function?

File: cli/installer/kubernetes.go [hunk 2]
```diff
@@ -21,7 +25,258 @@ var kubernetes = installer{
...
+cd $tmpdir/helm-chart
+helm dependency update
+
+helm upgrade --install demo . \
+  --namespace demo --create-namespace \
+  -f values.yaml \
+  --set image.tag=latest \
+  --set image.pullPolicy=Always \
+  --set postgres.auth.username=ashketchum,postgres.auth.password=squirtle123,postgres.auth.database=pokeshop \
+  --set rabbitmq.auth.username=guest,rabbitmq.auth.password=guest,rabbitmq.auth.erlangCookie=secretcookie \
```
Comment (mathnogueira) [inline cli/installer/kubernetes.go:153]: I think you forgot to set the jaeger config. I cannot get traces back.

**Review by mathnogueira — CHANGES_REQUESTED** [review]

File: cli/installer/kubernetes.go (hunk 1 above)
Comment (schoren) [reply cli/installer/kubernetes.go:71]: `execCmd` internally calls `_execCmd` and exists on error. `execCmdIgnoreErrors` calls `_execCmd` and ignores errors. I don't want to directly call `_execCmd`

File: cli/installer/kubernetes.go (hunk 2 above)
Comment (schoren) [reply cli/installer/kubernetes.go:162]: good catch. It's fixed now

Comment (mathnogueira) [reply cli/installer/kubernetes.go:162]: Screenshot from 2022-09-13 14-02-31
The Host is correct, but the port is set as the `namespace` instead.

Comment (schoren) [reply cli/installer/kubernetes.go:162]: Fixed!

**Review by mathnogueira — APPROVED** [review]
Great, everything works now! GJ

**Pull request merged**